packages/__docs__/webpack.config.js

```diff
@@ -53,7 +58,19 @@ const config = merge(baseConfig, {
     static: {
       directory: outputPath
     },
-    host: '0.0.0.0'
+    host: '0.0.0.0',
+    // Webpack reloads the page if these change when running in watch mode
+    watchFiles: ['markdown-and-sources-data.json', 'icons-data.json'],
+    // listen to changes do docs source files and rebuild the JSON if they change
```

Comment:
another key part. I use `chokidar` to watch for source code and markdown file changes, if this happens it will recompile the needed JSON and reload the app.

It does not recompile the metadata JSON and the icons JSON, to see changes in these one has to restart the app.